### ISSUE TITLE
Mouse completely disappeared after mini moved in turn-based mode

### ISSUE THREAD

**Issue by AussieMongrel** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#439]
**Was unable to view and had very limited use of the cursor after i picked up a mini and slightly pushed it into a wall while in turn-based mode**

was using the click, (Hold Shift) and arrow keys to place a mini against a wall then tried to use the mouse to reposition the angle it was facing before the cursor disappeared and the mini began to float in place.
was able to use hotkeys but was unable to access anything via the cursor although camera operation functioned it was very erratic.

**To Reproduce**
Steps to reproduce the behaviour:
1. create a bed of dungeon tiles.
2. Create a strip in which two walls (with ground tile) are back to back.
3. spawn and select mini then manual move space by space using shift and arrows keys to place alongside the edge of two walls back to back.
4. Use mouse to pick up mini (Burly, Cleric) while in turn-based mode and rotate slightly while pushing into a wall
5. Mini should stick in place and the cursor will disappear leaving you unable to select anything with it. (Although camera-based movements w3ith the mouse sort of work but wiggle and lose accuracy).
6. Alt-F4 to bring up exit menu seems to work in turns of bringing the cursor back into a usable state.

**Additional context**
I was streaming my screen with friends on Discord at the same time to show off the Beta,
.jpeg image wouldn't attach to log

**Comment by LuPro** [comment]
Probably related to #29

**Comment by AussieMongrel** [comment]
yeah, read through #29 and it seems to be the same bug although I was able to replicate mine again only once.
